src/main/java/io/xpring/payid/idiomatic/PayIdException.java

```diff
@@ -0,0 +1,38 @@
+package io.xpring.payid.idiomatic;
+
+/**
+ * Represents an exception thrown from PayID Components.
+ */
+public class PayIdException extends Exception {
```

Comment:
If `PayIdException` extended `PayIDException` then you could replace usages of `PayIDException` in the SDK with `PayIdException` and legacy clients would still work as expected and get a deprecation warning to catch the new exception instead.

Reply:
Never mind, that won't work because they use a different enum type.